azure-mgmt-containerinstance/src/main/java/com/microsoft/azure/management/containerinstance/ContainerGroup.java

```diff
@@ -0,0 +1,729 @@
+                 *   mapping is not supported.
+                 *
+                 * @param ports array of UDP ports to be exposed externally
+                 * @return the next stage of the definition
+                 */
+                WithPortsOrAttach<ParentT> withExternalUdpPorts(int... ports);
+
+                /**
+                 * Specifies the container's UDP port available to external clients.
+                 * <p>
+                 * A public IP address will be create to allow external clients to reach the containers within the group.
+                 * To enable external clients to reach a container within the group, you must expose the port on the
+                 *   IP address and from the container. Because containers within the group share a port namespace, port
+                 *   mapping is not supported.
+                 *
+                 * @param port UDP port to be exposed externally
+                 * @return the next stage of the definition
+                 */
+                WithPortsOrAttach<ParentT> withExternalUdpPort(int port);
+
+                /**
+                 * Specifies the container's ports are available to internal clients only (other container instances within the container group).
+                 * <p>
+                 * Containers within a group can reach each other via localhost on the ports that they have exposed,
+                 *   even if those ports are not exposed externally on the group's IP address.
+                 *
+                 * @param ports array of TCP ports to be exposed internally
+                 * @return the next stage of the definition
+                 */
+                WithPortsOrAttach<ParentT> withInternalPorts(int... ports);
+
+                /**
+                 * Specifies the container's port is available to internal clients only (other container instances within the container group).
+                 * <p>
+                 * Containers within a group can reach each other via localhost on the ports that they have exposed,
+                 *   even if those ports are not exposed externally on the group's IP address.
+                 *
+                 * @param port TCP port to be exposed internally
+                 * @return the next stage of the definition
+                 */
+                WithPortsOrAttach<ParentT> withInternalPort(int port);
+            }
+
+            /**
+             * The stage of the container instance definition allowing to specify the number of CPU cores.
+             * <p>
+             * The CPU cores can be specified as a fraction, i.e. 1.5 represents one and a half atomic CPU cores
+             *   will be assigned to this container instance.
+             *
+             * @param <ParentT> the stage of the parent definition to return to after attaching this definition
+             */
+            interface WithCPUCoreCount<ParentT> {
```

Comment:
wrong caps - 3+ letter acronyms are camel cased (Cpu)) - same in the method name.

Reply:
fixed